src/app/(pages)/(TwoColumn)/article/page.jsx

```diff
@@ -0,0 +1,68 @@
+import Pager from "@/components/common/Pager";
+import { getContentList } from "@/components/common/fetchData";
+import CardList from "@/components/ui/CardList";
+import Feature from "@/components/section/feature/Feature";
+import { getLabels } from "@/components/common/fetchData";
+
+import {
+  Banner,
+  Breadcrumb,
+  PageTitle,
+  TagArea,
+  TagKeyword,
+} from "@/components/common";
+
+export default async function Event({ searchParams }) {
+  const page = Number(
```

Comment:
@VortexExpansion 
pls use `useCallback()`

Reply:
@yabe-diverta san 

I have a concern here. To use useCallback I would have to force this page to be a client component whereas currently this is a server component. Would that not also be less efficient than before.

Secondly, I had some problems using useCallback here because the default value I provided was being re-rendered again and again in the useEffect leading to infinite console statements. 

I am attacking a txt file for `src/app/(pages)/(TwoColumn)/article/page.jsx` 
can you please replace this and check how to proceed ahead.

[page.txt](https://github.com/diverta/front_next_media/files/13971693/page.txt)